vita3k/codec/include/codec/state.h

```diff
@@ -236,6 +238,6 @@ struct PlayerState {
     ~PlayerState();
 };
 
-void convert_yuv_to_rgb(const uint8_t *yuv, uint8_t *rgba, uint32_t width, uint32_t height);
+void convert_yuv_to_rgb(const uint8_t *yuv, uint8_t *rgba, uint32_t width, uint32_t height, AVPixelFormat format);
```

Comment:
```suggestion
void convert_yuv_to_rgb(const uint8_t *yuv, uint8_t *rgba, uint32_t width, uint32_t height, bool is_yuv_444);
```
Or something similar, so that you don't have to link ffmpeg to the modules library for a single declaration.

Reply:
I know, i have thinks very some like that, i search ^^